internal/pkg/cli/svc_logs.go

```diff
@@ -179,6 +195,14 @@ func (o *svcLogsOpts) Execute() error {
 	if o.limit != 0 {
 		limit = aws.Int64(int64(o.limit))
 	}
+	if o.previous {
+		taskID, err := o.setPreviousStoppedTaskID()
+		if taskID != "" {
+			o.taskIDs = []string{taskID}
+		} else {
+			return err
+		}
+	}
 	err := o.logsSvc.WriteLogEvents(logging.WriteLogEventsOpts{
```

Comment:
because `taskIDs` is `nil`, is it going to print all _running_ task logs instead of not printing?

Reply:
Yes, exactly right.